mmv1/products/compute/api.yaml

```diff
@@ -15647,6 +15647,99 @@ objects:
         required: true
         update_verb: :POST
         update_url: 'projects/{{project}}/regions/{{region}}/targetHttpsProxies/{{name}}/setUrlMap'
+  - !ruby/object:Api::Resource
+    name: 'RegionTargetTcpProxy'
+    kind: 'compute#targetTcpProxy'
```

Comment:
This isn't used and can be removed

Reply:
`kind` is not used? Seems to be everywhere...

Reply:
It doesn't actually get used in the generated code. It's a relic of past implementations